src/test/java/org/phoebus/channelfinder/docker/ChannelFinderChannelsIT.java

```diff
@@ -131,6 +135,28 @@ public static void tearDownObjects() {
         channel_c1_owner_o2 = null;
     }
 
+    @AfterAll
+    public static void extractJacocoReport() {
+        // extract jacoco report from container file system
+        //     stop jvm to make data available
```

Comment:
This method is repeated in all the docker IT Tests. It could be written once in ITUtil and called with the arguement of the class simple name.